students/choltzman/lesson03/mailroom.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+from textwrap import dedent
+
+# putting variables outside a function like this is ugly, but oh well
+donors = [
+        {
+            "name": "William Gates, III",
+            "gifts": [1234.56, 12345.67]
+        },
+        {
+            "name": "Mark Zuckerberg",
+            "gifts": [9506.31, 8288.91, 1357.02]
+        },
+        {
+            "name": "Jeff Bezos",
+            "gifts": [1234567.89]
+        },
+        {
+            "name": "Paul Allen",
+            "gifts": [1, 2, 3, 4, 5, 6, 7, 8, 9]
+        }
+]
+
+
+def main():
+    """Prompt user for a choice of actions."""
+    # intro text to use for prompt
+    intro = dedent("""\
+
+    -----------------------
+    Choose an action:
+        1: Send a Thank You
+        2: Create a Report
+        q: Quit
+    -----------------------""")
+
+    while True:
+        print(intro)
+        inp = input("Select: ")
+        print("")
+        if inp == "1":
+            send_thanks()
+        elif inp == "2":
+            create_report()
+        elif inp == "q":
+            print("Exiting.")
+            exit(0)
```

Comment:
ideally you should put this into a separate function (this will make a difference in the lesson 04)  - for now it's ok.